ext/drwrap/drwrap.c

```diff
@@ -927,8 +927,6 @@ drwrap_init(void)
                       NULL);
     hashtable_init_ex(&wrap_table, WRAP_TABLE_HASH_BITS, HASH_INTPTR, false /*!str_dup*/,
                       false /*!synch*/, wrap_entry_free, NULL, NULL);
-    hashtable_init_ex(&call_site_table, CALL_SITE_TABLE_HASH_BITS, HASH_INTPTR,
```

Comment:
Don't forget the definition too: [here](https://github.com/DynamoRIO/dynamorio/blob/53577d796030c79790ca354a7e7ab4536bccd492/ext/drwrap/drwrap.c#L253)

Reply:
Wow that was removed in the original version of this.  After some git commit chain rearranging somehow it came back...a little disturbing.

Reply:
Gone now.